server/views/applications/application-origin.njk

```diff
@@ -0,0 +1,52 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "../partials/showErrorSummary.njk" import showErrorSummary %}
+{% from "../components/formFields/form-page-radios/macro.njk" import formPageRadios %}
+
+{% extends "../partials/layout.njk" %}
+
+{% block pageTitle %}
+    What type of application are you applying for?
```

Comment:
Not a big deal for MVP but I think we can come up with a question that doesn't repeat 'apply'. Maybe 'What type of application are you making?'